plugins/pencilblue/controllers/index.js

```diff
@@ -275,10 +275,13 @@ Index.prototype.renderContent = function(content, contentSettings, themeSettings
     ats.registerLocal('author_position', content.author_position ? content.author_position : '');
     ats.registerLocal('media_body_style', content.media_body_style ? content.media_body_style : '');
     ats.registerLocal('comments', function(flag, cb) {
-       if (isPage || !contentSettings.allow_comments) {
-           cb(null, '');
-           return;
-       }
+        if (isPage || !contentSettings.allow_comments) {
+           return (null, '');
+        }
+
+	if (!content.allow_comments) {
+	   return cb(null, '');
+	}
```

Comment:
It looks good to me.  I'll give @blakecallens a chance to comment before I merge.  

One more small change.  Could you please combine the if statements?

```
if (isPage || !contentSettings.allow_comments || !content.allow_comments) {
  return cb(null, '');
}
```